### ISSUE TITLE
Results of using different learning rates and more training epochs

### ISSUE THREAD

**Issue by KaiyangZhou** [mlfoundations/open_clip#22]
Very nice code!

I'm able to reproduce the zero-shot results on imagenet using cc3m (2,862,387 images in total for me) and the provided sample code.

I'd like to ask if you have tried different learning rates other than `1e-3` for `batch=128`? Would you be able to give more insights on how you ended up using `lr=1e-3`?

Also, I'd like to know if you have tried more training epochs, i.e. larger than 30. I'm curious if training with more epochs would help improve the zero-shot accuracy.

**Comment by carlini** [comment]
I tried once with 60 epochs, and the accuracy gain is maybe +1% ImageNet zero-shot top-5. We ran a few experiments with different learning rates, and 1e-3 was a good mix of (a) stable, and (b) nearly-optimal. When training with 5e-3 converged, it gave better results (maybe +1%, if that), but sometimes it exploded and so we stuck with 1e-3.

**Comment by KaiyangZhou** [comment]
I see. This information is super useful.

Thanks
